website/content/en/highlights/2022-05-23-0-23-0-upgrade-guide.md

```diff
@@ -12,6 +12,7 @@ badges:
 Vector's 0.23.0 release includes **breaking changes**:
 
 1. [The `.deb` package no longer enables and starts the Vector systemd service](#systemd-autostart)
+2. [`syslog` source and VRL's `parse_syslog` structured data fields made consistent](#parse-syslog)
```

Comment:
logfmt change not listed - maybe not removed from the TOC on the 0.22 guide as well?

Reply:
Oops, got greed with my cut/paste. Thanks for catching that.